packages/web-app-files/src/components/SideBar/Shares/FileLinks.vue

```diff
@@ -56,6 +56,7 @@
           >
             <name-and-copy :link="link" />
             <details-and-edit
+              v-if="!isLinkIndirect(link)"
```

Comment:
the <`details-and-edit`> component still needs to be displayed, since it contains the permissions. It has a `:modifiable` prop which can be set to false for indirect links. You could also consider moving the `via` information into this component and pass the necessary information down

Reply:
would remove the "via XY" completely since there is something similar in the `details-and-edit`